main/options/options.cc

```diff
@@ -630,7 +630,7 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                  "and `export` directives",
                                  cxxopts::value<vector<string>>(), "<name>");
     options.add_options(section)("packager-layers", "Valid layer names for packages, ordered lowest to highest.",
```

Comment:
Might be worth mentioning that this both: defines the layers and opts into the checks.